less/datepicker3.less

```diff
@@ -1,5 +1,5 @@
 .datepicker {
-	border-radius: 4px;
+	.border-radius(@border-radius-base);
```

Comment:
Well the mixin is used on the other file, I guessed it made sense to use it here as well.
Now I see that there is no such mixin on the build, gonna fix this.